src/Cards/styled.js

```diff
@@ -1,11 +1,173 @@
-import React from "react";
+import styled, { css, keyframes } from "styled-components";
 
-const Cards = () => {
-  return (
-    <div>
-      <p>Hello</p>
-    </div>
-  );
-};
+const flexCenter = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
 
-export default Cards;
+const glow = css`
+  box-shadow: 0 0 10px #f2f2f2, 0 0 40px #f2f2f2, 0 0 80px #f2f2f2;
+`;
+
+export const CardsContainer = styled.div`
+  ${flexCenter}
+  min-height: 100vh;
+  background: #ff4d52;
+`;
+
+//card
+
+export const Card = styled.div`
+  ${flexCenter}
+  background: #2f2b32;
+  color: #f2f2f2;
+  padding: 0px 0px 20px 0px;
+  width: 400px;
+  height: 525px;
+  box-shadow: 0 8px 6px -6px black;
+  justify-content: ${(props) => props.justify || "flex-start"};
+
+  @media screen and (max-width: 375px) {
+    width: 90vw;
+  }
+  @media screen and (max-width: 320px) {
+    width: 90vw;
+    height: 90vh;
+  }
+`;
+
+export const CardWindowBar = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: ${(props) => props.justify || "space-between"};
+  background-color: #f2f2f2;
+  width: 90%;
+  height: 40px;
+  color: #2f2b32;
+  padding: 0 5% 0 5%;
+  font-size: 25px;
+`;
+
+export const Question = styled.div`
+  font-size: 25px;
+  width: 90%;
+  margin: 25px 0;
+  text-align: left;
+`;
+
+export const Option = styled.div`
+  background: #f2f2f2;
+  color: #2f2b32;
+  width: 90%;
+  margin: 10px;
+  font-size: 15px;
+  font-weight: bold;
+  cursor: pointer;
+
+  &:hover {
+    ${glow}
+  }
+
+  ${(props) => {
+    return (
+      props.isActive &&
+      css`
+        box-shadow: 0 0 10px #f2f2f2, 0 0 40px #f2f2f2, 0 0 80px #f2f2f2;
+      `
+    );
+  }}
+
+  ${(props) => {
+    return (
+      props.isEmpty &&
+      css`
+        animation: ${shake} 1s cubic-bezier(0.36, 0.07, 0.19, 0.97) both;
+      `
+    );
+  }}
+
+  ${(props) => {
+    return (
+      props.isCorrect === "red" &&
```

Comment:
`isCorrect` should return true or false. So rather than have it return "red" or "green", this logic should check if it's true or false. If it's true, then set the background color to green. Else, set it to red.